shacl2md.py

```diff
@@ -157,18 +150,7 @@ def copy(self):
         return copy_class
 
     def to_dict(self):
-        return {
-            "lang": self.lang,
-            "iri": self.iri,
-            "shortname": self.shortname,
-            "label": self.label,
-            "description": self.description,
-            "properties": [prop.to_dict() for prop in self.properties],
```

Comment:
waar worden deze properties dan opgehaald als je gebruik maakt van `o.__dict__`?

Reply:
Door de `default` in `json.dumps()` is voor waarden waar serialisatie waarde voor gegeven kan worden (bijvoorbeeld een andere klasse als de properties) en daarvan neemt hij dus de `__dict__`.

Reply:
Die `default` van de `json.dumps` functie zal elk object dat niet kan geserialiseerd worden gebruiken. Die werkt blijkbaar dus ook recursief (dus op class in class).